src/components/footer/toolbar-draw.vue

```diff
@@ -22,52 +24,63 @@ useDraw()
       <li>
         <button-text
           :label="t('Draw Point', { ns: 'client' })"
-          :active="drawStateActive === 'drawPoint'"
-          @click="() => toggleActiveState('drawPoint')"
+          :active="
+            drawStateActive === 'drawPoint' || editStateActive === 'editPoint'
```

Comment:
maybe this can be simplified if the feature state has 2 parts, so we only have to check one type
- feature type
- feature context (draw, edit, etc.)

Reply:
Indeed, here an object would be more elegant.